WordPress/Classes/Utility/Editor/GutenbergOnboardingRollout.swift

```diff
@@ -1,9 +1,26 @@
 /// This structs helps encapsulate logic related to Gutenberg editor onboarding rollout phases.
 ///
 struct GutenbergOnboardingRollout {
-    private let phasePercentage = 0
+    private let phasePercentage = 50
 
     func isUserIdInPhaseRolloutPercentage(_ userId: Int) -> Bool {
-        return userId % 100 >= (100 - phasePercentage) || BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
+        return convertUserIdToRank(userId: String(userId)) >= (100 - phasePercentage) || BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
```

Comment:
nitpick: It might be good to name this something other than userId because it could be easily misinterpreted as always being a WP.com user ID, when in fact for non-WP.com users it's a UUID().hashValue value.
Maybe, uniqueUserIdentifier could also work, just to help distinguish it from WP.com user IDs.

Reply:
Updated in 4f623aa87923b37b99df26300b19064988ceb4d1